src/components/shared/SideNavigation.tsx

```diff
@@ -39,15 +40,17 @@ const SideNavigation: FC<ISideNavigationProps> = ({ navItems }) => {
   );
 
   return (
-    <HDSSideNavigation
-      id="side-navigation"
-      toggleButtonLabel={t('nav.navigateToForm')}
-      className="side-nav"
-    >
-      {hdsNavItems.map((s) => (
-        <HDSSideNavigation.MainLevel {...s} key={s.href} />
-      ))}
-    </HDSSideNavigation>
+    <>
```

Comment:
Is this fragment component needed as wrapper here?

Reply:
Fixed